pkgs/development/tools/gdtoolkit/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, python3 }:
+
+with python3.pkgs;
+
+buildPythonApplication rec {
+  pname = "gdtoolkit";
+  version = "3.2.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "3+zr1JcCt8HPwDG0j+ttnzGOhvUmaz+qzbE2wGyT86M=";
+  };
+
+  buildInputs = [ lark-parser docopt pyyaml setuptools ];
+  propagatedBuildInputs = [ lark-parser docopt pyyaml setuptools ];
+
+  preConfigure = ''
+    sed -i 's/lark-parser==0.8.0/lark-parser>=0.8.0/' setup.py
+  '';
+
+  doCheck = false;
+
+  meta = with lib; {
+    homepage = "https://github.com/Scony/godot-gdscript-toolkit";
+    description =
+      "Independent set of GDScript tools - parser, linter and formatter";
+    license = licenses.mit;
+    platforms = platforms.linux;
```

Comment:
Seems to work fine for me on darwin too so you can probably safely change this to `platforms.unix`.